File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/sqlserver/SpatialTypes.scala
package dev.typr.foundations.docs.sqlserver
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*


import com.microsoft.sqlserver.jdbc.{Geography, Geometry}

@SuppressWarnings(Array("unused"))
object SpatialTypes:
  //start
  val geoType: SqlServerType[Geography] = SqlServerTypes.geography
  val geomType: SqlServerType[Geometry] = SqlServerTypes.geometry
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/sqlserver/SpatialTypes.java
package dev.typr.foundations.docs.sqlserver;

import com.microsoft.sqlserver.jdbc.Geography;
import com.microsoft.sqlserver.jdbc.Geometry;
import dev.typr.foundations.SqlServerType;
import dev.typr.foundations.SqlServerTypes;

@SuppressWarnings("unused")
public class SpatialTypes {
  // start
  SqlServerType<Geography> geoType = SqlServerTypes.geography;
  SqlServerType<Geometry> geomType = SqlServerTypes.geometry;
  // stop
}
